cmd/applyconfig/applyconfig_test.go

```diff
@@ -64,47 +69,181 @@ func TestIsStandardConfig(t *testing.T) {
 	}
 }
 
-func TestMakeOcArgs(t *testing.T) {
+func TestOcApply(t *testing.T) {
 	testCases := []struct {
 		name string
 
 		path string
 		user string
 		dry  bool
 
-		expected []string
+		expected string
 	}{
 		{
 			name:     "no user, not dry",
 			path:     "/path/to/file",
-			expected: []string{"apply", "-f", "/path/to/file"},
+			expected: "oc apply -f /path/to/file",
 		},
 		{
 			name:     "no user, dry",
 			path:     "/path/to/different/file",
 			dry:      true,
-			expected: []string{"apply", "-f", "/path/to/different/file", "--dry-run"},
+			expected: "oc apply -f /path/to/different/file --dry-run",
 		},
 		{
 			name:     "user, dry",
 			path:     "/path/to/file",
 			dry:      true,
 			user:     "joe",
-			expected: []string{"apply", "-f", "/path/to/file", "--dry-run", "--as", "joe"},
+			expected: "oc apply -f /path/to/file --dry-run --as joe",
 		},
 		{
 			name:     "user, not dry",
 			path:     "/path/to/file",
 			user:     "joe",
-			expected: []string{"apply", "-f", "/path/to/file", "--as", "joe"},
+			expected: "oc apply -f /path/to/file --as joe",
 		},
 	}
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			args := makeOcArgs(tc.path, tc.user, tc.dry)
-			if !reflect.DeepEqual(args, tc.expected) {
-				t.Errorf("Expected '%v', got '%v'", tc.expected, args)
+			cmd := makeOcApply(tc.path, tc.user, tc.dry)
+			command := strings.Join(cmd.Args, " ")
```

Comment:
This fails to test an errant `Command` with one arg like `--foo bar` (where we would want two args, `--foo` `bar`)

Reply:
Addressed everywhere